pkg/webhooks/provisioning/v1alpha1/validator.go

```diff
@@ -97,7 +97,7 @@ func (v *Validator) validateLabels(ctx context.Context, spec *v1alpha1.Provision
 			}
 		}
 	}
-	return nil
+	return v.CloudProvider.CapacityFor(spec).ValidateLabels(spec.Labels)
```

Comment:
This might be not a problem, but thought I'd ask. It does seem potentially a bit odd to pass in `spec.Labels` given that the cloud provider already has a copy of the entire `ProvisionerSpec` so could just access the labels from there? Maybe it would make more sense for the cloud provider to just have a chance to validate the ProvisionerSpec?

Reply:
💯